src/Action_Hbond.cpp

```diff
@@ -76,6 +80,9 @@ Action::RetType Action_Hbond::Init(ArgList& actionArgs, ActionInit& init, int de
   useAtomNum_ = actionArgs.hasKey("printatomnum");
   acut_ = actionArgs.getKeyDouble("angle",135.0);
   noIntramol_ = actionArgs.hasKey("nointramol");
+  if (actionArgs.hasKey("atomindex")) {
+    AHDIndices_ = (DataSet_integer*) init.DSL().AddSet(DataSet::INTEGER, "atomindex");
```

Comment:
but the order of the indices does not match to cpptraj's label for each hbond.

For example:

this is the indices if specifying `atomindex`
```python
[[ 50   3   0]
 [  9  14  13]
 [ 89  48  47]
 ..., 
 [167 212 211]
 [170 212 211]
 [170 213 211]]
```

vs the indices from parsing cpptraj's hbond output: 'SER_1@OG-TRP_2@N-H', 'GLU_5@O-THR_3@OG1-HG1', ...

```python
[9, 14, 13], [89, 48, 47], [50, 3, 0], [103, 112, 111], [47, 100, 99], [36, 52, 51], [167, 130, 127], [170, 212, 211], [170, 168, 167], [167, 212, 211], [170, 213, 211], [87, 91, 90], [110, 134, 133]]
```